spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/cloud_manage_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  context ".raw_connect" do
+    let(:config) do
+      {
+        :provider               => "IbmCloud",
+        :IbmCloud_project         => "project",
+        :IbmCloud_json_key_string => "encrypted",
+        :app_name               => Vmdb::Appliance.PRODUCT_NAME,
+        :app_version            => Vmdb::Appliance.VERSION,
+        :Ibm_client_options  => { :proxy_url => "proxy_uri" },
+      }
+    end
+  end
+  it ".ems_type" do
+    expect(described_class.ems_type).to eq('ibm_cloud_power_virtual_servers')
+  end
+
+  it ".description" do
+    expect(described_class.description).to eq('IBM Cloud Power Systems Virtual Server')
+  end
+
+  it "does not create orphaned network_manager" do
+    ems = FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud)
+    same_ems = ExtManagementSystem.find(ems.id)
+
+    ems.destroy
+    expect(ExtManagementSystem.count).to eq(0)
+
+    same_ems.save!
+    expect(ExtManagementSystem.count).to eq(0)
+  end
```

Comment:
The specs this looks like it was based on typically also ensure that when the zone is changed on the cloud manager we also change the zone on the child managers (e.g. `it "moves the network_manager to the same zone as the cloud_manager" do`) which is a good test, can you add that as well?